src/services/pathContents/utils.ts

```diff
@@ -14,7 +14,7 @@ export function extractCoverageFromResponse(
   if (!coverageFile) return null
   const lineWithCoverage = keyBy(coverageFile?.coverage, 'line')
   const fileCoverage = mapValues(lineWithCoverage, 'coverage')
-  const coverageTotal = coverageFile?.totals?.coverage
```

Comment:
was this a bug fix too?

Reply:
Nope, i had originally aliased the field in GQL, this is me removing it now and updating it, with TS makes it easier to find the errors

Reply:
nice! love it